src/popup/components/Header.vue

```diff
@@ -248,6 +248,11 @@ export default {
     margin-right: 8px;
   }
 
+  .btn-close {
```

Comment:
This class does not exist in the scope of the component. If you want to style something within the scope of the component you need to add additional class to this element.